lib/src/executables/boilerplate_upgrade.dart

```diff
@@ -124,8 +147,8 @@ void main(List<String> args) {
       GeneratedPartDirectiveIgnoreRemover(),
       FactoryIgnoreCommentMover(),
     ].map((s) => Ignoreable(s)),
-    args: parsedArgs.rest,
     defaultYes: true,
+    args: [],
```

Comment:
Can we not pass anything for `args`, or do we need an empty list?

Reply:
We need an empty list. Null throws